cms/static/js/factories/videos_index.js

```diff
@@ -10,10 +10,26 @@ define(
             uploadButton,
             previousUploads
         ) {
+            function refreshList() {
+                $.ajax({
+                    url: postUrl,
+                    contentType: 'application/json',
+                    dataType: 'json',
+                    type: 'GET'
+                }).done(function(responseData) {
+                    var updatedCollection = new Backbone.Collection(responseData.videos),
```

Comment:
I am not very sure if that's possible without changing the routine called on the server. I will check it up.